doc/source/theory/unres_cfd-dem/unres_cfd-dem.rst

```diff
@@ -0,0 +1,150 @@
+* :math:`v_i` is the velocity vector;
+* :math:`\mathbf{f}_{c,ij}` are the contact forces between particles :math:`i` and :math:`j` (detailed in the DEM section of this guide);
+* :math:`\mathbf{f}_{nc,ij}` are the non-contact forces between particles :math:`i` and :math:`j`, such as lubrication `[3] <https://doi.org/10.1002/aic.690400418>`_.
+* :math:`\mathbf{f}_{pf,i}` is the force exerted by the surrounding fluid over particle :math:`i`;
+* :math:`\mathbf{f}_{g,i}` is the gravity force;
+* :math:`I_i` is the momentum of inertia;
+* :math:`\mathbf{\omega}_i` is the angular velocity;
+* :math:`\mathbf{M}_{c,ij}` is the torque between particles :math:`i` and :math:`j`;
+* :math:`\mathbf{M}_{r,ij}` is the rolling friction between particles :math:`i` and :math:`j`;
+* :math:`\mathbf{M}_{c,iw}` is the torque between particle :math:`i` and walls :math:`w`;
+* :math:`\mathbf{M}_{c,iw}` is the rolling friction between particle :math:`i` and walls :math:`w`;
+
+Apart from :math:`\mathbf{f}_{pf,i}`, all the other terms of the previous equations are detailed in the DEM section of this theory guide (**under construction**). The momentum transport between phases :math:`\mathbf{f}_{pf,i}` can be written as:
+
+.. math::
+    \mathbf{f}_{pf,i} = \mathbf{f}_{\nabla p,i} + \mathbf{f}_{\nabla \cdot \mathbf{\tau},i} + \mathbf{f}_{d,i} + \mathbf{f}_{Ar,i} + \mathbf{f}_{g,i} + \mathbf{f}''_{i}
+
+where:
+
+* :math:`\mathbf{f}_{\nabla p,i}` is the force due to the pressure gradient;
+* :math:`\mathbf{f}_{\nabla \cdot \tau,i}` is the force due to the shear stress;
+* :math:`\mathbf{f}_{d,i}` is the drag force;
+* :math:`\mathbf{f}_{Ar,i}` is the buoyant (Archimedes) force;
+* :math:`\mathbf{f}_{g,i}` is the force due to gravity;
+* :math:`\mathbf{f}''_{i}` are the remaining forces, including virtual mass, Basset, Lift, and Magnus (currently not implemented in Lethe).
+
+.. note::
+    Since pressure in Lethe does not account for the hydrostatic pressure, i.e., the gravity term is not taken into account in the Navier-Stokes equations (see :doc:`../fluid_dynamics/navier-stokes`), we explicitly insert :math:`\mathbf{f}_{Ar,i}` in :math:`\mathbf{f}_{pf,i}`.
+
+In unresolved CFD-DEM drag is calculated using correlations (frequently called drag models). The drag models implemented in Lethe are described in the `unresolved CFD-DEM parameters guide <https://lethe-cfd.github.io/lethe/parameters/unresolved_cfd-dem/cfd_dem.html>`_.
+
+Volume Average Navier-Stokes
+-----------------------------
+
+Since we represent the fluid at a meso-scale, the quantities calculated for the subdomains are averages among its volume. Additionally, as the volume of fluid is a fraction of the subdomain, the porosity (or void fraction) is taken into account. To do this, we apply the Volume Average Navier-Stokes (VANS) equations to represent the fluid phase. Mainly, the VANS equations are presented in two different formulations, so called Model A (or Set II) and Model B (or Set I).
```

Comment:
```suggestion
Since we represent the fluid at a meso-scale, the quantities calculated for the subdomains are averages among its volume. Additionally, as the volume of fluid is a fraction of the subdomain, the porosity (or void fraction) is taken into account. To do this, we apply the Volume Average Navier-Stokes (VANS) equations to represent the fluid phase. Mainly, the VANS equations are presented in two different formulations, so called Model A (or Set II) and Model B (or Set I).
```
What is meant by subdomain here? You mean cells or you mean phases? This is not clear.

Reply:
I changed 'subdomains' for 'cells'. I think it is more clear now. What do you think?